driloader/browser_detection.py

```diff
@@ -177,14 +178,19 @@ def _run_command(command):
             command_array = command
 
         try:
-            cmd_result = subprocess.run(command_array, stdout=subprocess.PIPE)
+            if int(sys.version[0]) == 3 and int(sys.version[2]) < 5:
```

Comment:
I suggest you to create a new function to run external commands. So we can encapsulate this logic in one place.

If you want use `if` statement in this case, you can change it to something like that:

```
if hasattr(subprocess, 'run'):
    cmd_result = subprocess.run(command_array, stdout=subprocess.PIPE)
else:
    cmd_result = subprocess.check_output(command_array)
```

Or instead you can use the EAFP approach:

```
try:
    cmd_result = subprocess.run(command_array, stdout=subprocess.PIPE)
except:
    cmd_result = subprocess.check_output(command_array)
```

I recommend the second choice.